### PR TITLE
✨ PIC-1197 return correct probation status

### PR DESCRIPTION
**Pull request by chris-faulkner-mojdt**
Also remove feature flag which allowed support for replaced community API calls

### CODE REVIEW

File: src/main/resources/application.yml [hunk 1]
```diff
@@ -127,3 +127,7 @@ offender-service:
   assessment:
     # Comma-separated, all upper case
     included-statuses: "COMPLETE"
+
+feature:
+  flags:
+    use-community-api-for-probation-status: true
```
Comment (jhevans) [inline src/main/resources/application.yml:133]: This can be removed now can't it?

Comment (jhevans) [inline src/main/resources/application.yml:133]:
```suggestion
```

**Review by jhevans — CHANGES_REQUESTED** [review]
Think there's some config that can go, have suggested the changes in line so hopefully you can just do it in Github

**Comment by chris-faulkner-mojdt** [comment]
> Think there's some config that can go, have suggested the changes in line so hopefully you can just do it in Github

Not sure how to do that

File: src/main/resources/application.yml (hunk 1 above)
Comment (chris-faulkner-mojdt) [reply src/main/resources/application.yml]: Done

**Review by jhevans — APPROVED** [review]

**Pull request merged**